### PR TITLE
test/crimson/cbt: increase stddev variance to 2

### PR DESCRIPTION
**Pull request by tchaikov**
as per Mark requests.

Signed-off-by: Kefu Chai <kchai@redhat.com>

## Checklist
- [ ] References tracker ticket
- [ ] Updates documentation if necessary
- [ ] Includes tests for new functionality or reproducer for bug

---

<details>
<summary>Show available Jenkins commands</summary>

- `jenkins retest this please`
- `jenkins test classic perf`
- `jenkins test crimson perf`
- `jenkins test signed`
- `jenkins test make check`
- `jenkins test make check arm64`
- `jenkins test submodules`
- `jenkins test dashboard`
- `jenkins test dashboard backend`
- `jenkins test docs`
- `jenkins render docs`
- `jenkins test ceph-volume all`
- `jenkins test ceph-volume tox`

</details>

### CODE REVIEW

File: src/test/crimson/cbt/radosbench_4K_read.yaml [hunk 1]
```diff
@@ -18,7 +18,7 @@ tasks:
         acceptable:
           bandwidth: '(or (greater) (near 0.05))'
           iops_avg: '(or (greater) (near 0.05))'
-          iops_stddev: '(or (less) (near 0.20))'
+          iops_stddev: '(or (less) (near 0.50))'
```
Comment (rzarzynski) [inline src/test/crimson/cbt/radosbench_4K_read.yaml:21]: Maybe we should consider dropping `iops_stddev` entirely.

**Comment by tchaikov** [comment]
```
15:41:24 - DEBUG    - cbt      - Nodes : incerta04.front.sepia.ceph.com
15:41:24 - WARNING  - cbt      - prefill/incerta04.front.sepia.ceph.com/0: bandwidth: (or (greater) (near 0.05)):: 27.9053/31.8056  => rejected
15:41:24 - WARNING  - cbt      - prefill/incerta04.front.sepia.ceph.com/0: iops_avg: (or (greater) (near 0.05)):: 7143.0/8142.0  => rejected
15:41:24 - INFO     - cbt      - prefill/incerta04.front.sepia.ceph.com/0: iops_stddev: (or (less) (near 0.50)):: 300.215/252.114  => accepted
15:41:24 - WARNING  - cbt      - prefill/incerta04.front.sepia.ceph.com/0: latency_avg: (or (less) (near 0.05)):: 0.00220615/0.00195175  => rejected
15:41:24 - WARNING  - cbt      - prefill/incerta04.front.sepia.ceph.com/1: bandwidth: (or (greater) (near 0.05)):: 29.2199/31.2244  => rejected
15:41:24 - WARNING  - cbt      - prefill/incerta04.front.sepia.ceph.com/1: iops_avg: (or (greater) (near 0.05)):: 7480.0/7993.0  => rejected
15:41:24 - INFO     - cbt      - prefill/incerta04.front.sepia.ceph.com/1: iops_stddev: (or (less) (near 0.50)):: 298.992/215.296  => accepted
15:41:24 - WARNING  - cbt      - prefill/incerta04.front.sepia.ceph.com/1: latency_avg: (or (less) (near 0.05)):: 0.00212452/0.00196675  => rejected
15:41:24 - INFO     - cbt      - rand/incerta04.front.sepia.ceph.com/0: bandwidth: (or (greater) (near 0.05)):: 246.234/252.905  => accepted
15:41:24 - INFO     - cbt      - rand/incerta04.front.sepia.ceph.com/0: iops_avg: (or (greater) (near 0.05)):: 63036.0/64743.0  => accepted
15:41:24 - WARNING  - cbt      - rand/incerta04.front.sepia.ceph.com/0: iops_stddev: (or (less) (near 0.50)):: 1480.53/965.09  => rejected
15:41:24 - INFO     - cbt      - rand/incerta04.front.sepia.ceph.com/0: latency_avg: (or (less) (near 0.05)):: 0.000250633/0.000244159  => accepted
15:41:24 - INFO     - cbt      - rand/incerta04.front.sepia.ceph.com/1: bandwidth: (or (greater) (near 0.05)):: 247.675/249.351  => accepted
15:41:24 - INFO     - cbt      - rand/incerta04.front.sepia.ceph.com/1: iops_avg: (or (greater) (near 0.05)):: 63404.0/63833.0  => accepted
15:41:24 - INFO     - cbt      - rand/incerta04.front.sepia.ceph.com/1: iops_stddev: (or (less) (near 0.50)):: 692.832/868.106  => accepted
15:41:24 - INFO     - cbt      - rand/incerta04.front.sepia.ceph.com/1: latency_avg: (or (less) (near 0.05)):: 0.000249233/0.000247789  => accepted
15:41:24 - WARNING  - cbt      - 7 tests failed out of 16
```
probably i should use "jenkins, roll the dice again" as the trigger phrase..

**Comment by tchaikov** [comment]
jenkins test classic perf

File: src/test/crimson/cbt/radosbench_4K_read.yaml (hunk 1 above)
Comment (markhpc) [reply src/test/crimson/cbt/radosbench_4K_read.yaml:21]: Do you mean just the requirement to pass, or stop recording it?  I think we should still record it, and perhaps in the future we'll be able to narrow it down.

**Comment by markhpc** [comment]
@tch

> ```
> 15:41:24 - DEBUG    - cbt      - Nodes : incerta04.front.sepia.ceph.com
> 15:41:24 - WARNING  - cbt      - prefill/incerta04.front.sepia.ceph.com/0: bandwidth: (or (greater) (near 0.05)):: 27.9053/31.8056  => rejected
> 15:41:24 - WARNING  - cbt      - prefill/incerta04.front.sepia.ceph.com/0: iops_avg: (or (greater) (near 0.05)):: 7143.0/8142.0  => rejected
> 15:41:24 - INFO     - cbt      - prefill/incerta04.front.sepia.ceph.com/0: iops_stddev: (or (less) (near 0.50)):: 300.215/252.114  => accepted
> 15:41:24 - WARNING  - cbt      - prefill/incerta04.front.sepia.ceph.com/0: latency_avg: (or (less) (near 0.05)):: 0.00220615/0.00195175  => rejected
> 15:41:24 - WARNING  - cbt      - prefill/incerta04.front.sepia.ceph.com/1: bandwidth: (or (greater) (near 0.05)):: 29.2199/31.2244  => rejected
> 15:41:24 - WARNING  - cbt      - prefill/incerta04.front.sepia.ceph.com/1: iops_avg: (or (greater) (near 0.05)):: 7480.0/7993.0  => rejected
> 15:41:24 - INFO     - cbt      - prefill/incerta04.front.sepia.ceph.com/1: iops_stddev: (or (less) (near 0.50)):: 298.992/215.296  => accepted
> 15:41:24 - WARNING  - cbt      - prefill/incerta04.front.sepia.ceph.com/1: latency_avg: (or (less) (near 0.05)):: 0.00212452/0.00196675  => rejected
> 15:41:24 - INFO     - cbt      - rand/incerta04.front.sepia.ceph.com/0: bandwidth: (or (greater) (near 0.05)):: 246.234/252.905  => accepted
> 15:41:24 - INFO     - cbt      - rand/incerta04.front.sepia.ceph.com/0: iops_avg: (or (greater) (near 0.05)):: 63036.0/64743.0  => accepted
> 15:41:24 - WARNING  - cbt      - rand/incerta04.front.sepia.ceph.com/0: iops_stddev: (or (less) (near 0.50)):: 1480.53/965.09  => rejected
> 15:41:24 - INFO     - cbt      - rand/incerta04.front.sepia.ceph.com/0: latency_avg: (or (less) (near 0.05)):: 0.000250633/0.000244159  => accepted
> 15:41:24 - INFO     - cbt      - rand/incerta04.front.sepia.ceph.com/1: bandwidth: (or (greater) (near 0.05)):: 247.675/249.351  => accepted
> 15:41:24 - INFO     - cbt      - rand/incerta04.front.sepia.ceph.com/1: iops_avg: (or (greater) (near 0.05)):: 63404.0/63833.0  => accepted
> 15:41:24 - INFO     - cbt      - rand/incerta04.front.sepia.ceph.com/1: iops_stddev: (or (less) (near 0.50)):: 692.832/868.106  => accepted
> 15:41:24 - INFO     - cbt      - rand/incerta04.front.sepia.ceph.com/1: latency_avg: (or (less) (near 0.05)):: 0.000249233/0.000247789  => accepted
> 15:41:24 - WARNING  - cbt      - 7 tests failed out of 16
> ```
> 
> probably i should use "jenkins, roll the dice again" as the trigger phrase..

hah! That would be a very apt description I think.  :smile:   I think we probably have to start out with very loose requirements at first and figure out what we can narrow down over time.

**Comment by tchaikov** [comment]
@markhpc i am not sure what the right approach is ... my biggest concern is that the performance could be degraded by a random PR which is merged because we are very tolerant. and when we narrow the the requirements down, it might be way too late.

**Comment by tchaikov** [comment]
```
16:04:52 - DEBUG    - cbt      - Nodes : incerta04.front.sepia.ceph.com
16:04:52 - INFO     - cbt      - prefill/incerta04.front.sepia.ceph.com/0: bandwidth: (or (greater) (near 0.05)):: 56.6441/55.491  => accepted
16:04:52 - INFO     - cbt      - prefill/incerta04.front.sepia.ceph.com/0: iops_avg: (or (greater) (near 0.05)):: 14500.0/14205.0  => accepted
16:04:52 - INFO     - cbt      - prefill/incerta04.front.sepia.ceph.com/0: iops_stddev: (or (less) (near 0.50)):: 405.641/330.536  => accepted
16:04:52 - INFO     - cbt      - prefill/incerta04.front.sepia.ceph.com/0: latency_avg: (or (less) (near 0.05)):: 0.00110145/0.00112421  => accepted
16:04:52 - WARNING  - cbt      - prefill/incerta04.front.sepia.ceph.com/1: bandwidth: (or (greater) (near 0.05)):: 53.3672/68.4813  => rejected
16:04:52 - WARNING  - cbt      - prefill/incerta04.front.sepia.ceph.com/1: iops_avg: (or (greater) (near 0.05)):: 13661.0/17531.0  => rejected
16:04:52 - INFO     - cbt      - prefill/incerta04.front.sepia.ceph.com/1: iops_stddev: (or (less) (near 0.50)):: 473.029/461.434  => accepted
16:04:52 - WARNING  - cbt      - prefill/incerta04.front.sepia.ceph.com/1: latency_avg: (or (less) (near 0.05)):: 0.00116924/0.000910561  => rejected
16:04:52 - WARNING  - cbt      - rand/incerta04.front.sepia.ceph.com/0: bandwidth: (or (greater) (near 0.05)):: 224.244/241.664  => rejected
16:04:52 - WARNING  - cbt      - rand/incerta04.front.sepia.ceph.com/0: iops_avg: (or (greater) (near 0.05)):: 57406.0/61865.0  => rejected
16:04:52 - INFO     - cbt      - rand/incerta04.front.sepia.ceph.com/0: iops_stddev: (or (less) (near 0.50)):: 2641.96/4233.14  => accepted
16:04:52 - WARNING  - cbt      - rand/incerta04.front.sepia.ceph.com/0: latency_avg: (or (less) (near 0.05)):: 0.000275652/0.000255626  => rejected
16:04:52 - INFO     - cbt      - rand/incerta04.front.sepia.ceph.com/1: bandwidth: (or (greater) (near 0.05)):: 226.188/218.363  => accepted
16:04:52 - INFO     - cbt      - rand/incerta04.front.sepia.ceph.com/1: iops_avg: (or (greater) (near 0.05)):: 57904.0/55901.0  => accepted
16:04:52 - INFO     - cbt      - rand/incerta04.front.sepia.ceph.com/1: iops_stddev: (or (less) (near 0.50)):: 2011.25/2676.23  => accepted
16:04:52 - INFO     - cbt      - rand/incerta04.front.sepia.ceph.com/1: latency_avg: (or (less) (near 0.05)):: 0.000273249/0.000283003  => accepted
16:04:52 - WARNING  - cbt      - 6 tests failed out of 16
```

**Comment by tchaikov** [comment]
jenkins test classic perf

**Comment by markhpc** [comment]
> @markhpc i am not sure what the right approach is ... my biggest concern is that the performance could be degraded by a random PR which is merged because we are very tolerant. and when we narrow the the requirements down, it might be way too late.

That's possible, but it would still be a better situation than we have now as we could at least go back and look for trends in the performance data to narrow down when it happened even if it's noisy.  I don't think we'll ever get to the point where we can avoid all false positives or false negatives.  It's going to be a balancing act.

**Comment by tchaikov** [comment]
jenkins test classic perf

**Comment by tchaikov** [comment]
```
03:55:21 - DEBUG    - cbt      - Nodes : incerta04.front.sepia.ceph.com
03:55:21 - INFO     - cbt      - prefill/incerta04.front.sepia.ceph.com/0: bandwidth: (or (greater) (near 0.05)):: 59.9503/57.0622  => accepted
03:55:21 - INFO     - cbt      - prefill/incerta04.front.sepia.ceph.com/0: iops_avg: (or (greater) (near 0.05)):: 15347.0/14607.0  => accepted
03:55:21 - WARNING  - cbt      - prefill/incerta04.front.sepia.ceph.com/0: iops_stddev: (or (less) (near 0.50)):: 258.883/74.9289  => rejected
03:55:21 - INFO     - cbt      - prefill/incerta04.front.sepia.ceph.com/0: latency_avg: (or (less) (near 0.05)):: 0.00104052/0.00109322  => accepted
03:55:21 - INFO     - cbt      - prefill/incerta04.front.sepia.ceph.com/1: bandwidth: (or (greater) (near 0.05)):: 63.2498/55.3239  => accepted
03:55:21 - INFO     - cbt      - prefill/incerta04.front.sepia.ceph.com/1: iops_avg: (or (greater) (near 0.05)):: 16191.0/14162.0  => accepted
03:55:21 - WARNING  - cbt      - prefill/incerta04.front.sepia.ceph.com/1: iops_stddev: (or (less) (near 0.50)):: 434.037/69.2122  => rejected
03:55:21 - INFO     - cbt      - prefill/incerta04.front.sepia.ceph.com/1: latency_avg: (or (less) (near 0.05)):: 0.000986114/0.00112765  => accepted
03:55:21 - WARNING  - cbt      - rand/incerta04.front.sepia.ceph.com/0: bandwidth: (or (greater) (near 0.05)):: 232.532/251.842  => rejected
03:55:21 - WARNING  - cbt      - rand/incerta04.front.sepia.ceph.com/0: iops_avg: (or (greater) (near 0.05)):: 59528.0/64471.0  => rejected
03:55:21 - INFO     - cbt      - rand/incerta04.front.sepia.ceph.com/0: iops_stddev: (or (less) (near 0.50)):: 1750.55/3157.23  => accepted
03:55:21 - WARNING  - cbt      - rand/incerta04.front.sepia.ceph.com/0: latency_avg: (or (less) (near 0.05)):: 0.000265736/0.000245256  => rejected
03:55:21 - INFO     - cbt      - rand/incerta04.front.sepia.ceph.com/1: bandwidth: (or (greater) (near 0.05)):: 223.266/219.347  => accepted
03:55:21 - INFO     - cbt      - rand/incerta04.front.sepia.ceph.com/1: iops_avg: (or (greater) (near 0.05)):: 57156.0/56152.0  => accepted
03:55:21 - INFO     - cbt      - rand/incerta04.front.sepia.ceph.com/1: iops_stddev: (or (less) (near 0.50)):: 2078.81/1425.67  => accepted
03:55:21 - INFO     - cbt      - rand/incerta04.front.sepia.ceph.com/1: latency_avg: (or (less) (near 0.05)):: 0.000276879/0.000281809  => accepted
03:55:21 - WARNING  - cbt      - 5 tests failed out of 16
```

**Comment by tchaikov** [comment]
jenkins test classic perf

**Comment by tchaikov** [comment]
```
04:37:52 - INFO     - cbt      - prefill/incerta04.front.sepia.ceph.com/0: bandwidth: (or (greater) (near 0.05)):: 58.8936/53.3138  => accepted
04:37:52 - INFO     - cbt      - prefill/incerta04.front.sepia.ceph.com/0: iops_avg: (or (greater) (near 0.05)):: 15076.0/13648.0  => accepted
04:37:52 - INFO     - cbt      - prefill/incerta04.front.sepia.ceph.com/0: iops_stddev: (or (less) (near 0.50)):: 13.6137/329.638  => accepted
04:37:52 - INFO     - cbt      - prefill/incerta04.front.sepia.ceph.com/0: latency_avg: (or (less) (near 0.05)):: 0.00105895/0.00117027  => accepted
04:37:52 - INFO     - cbt      - prefill/incerta04.front.sepia.ceph.com/1: bandwidth: (or (greater) (near 0.05)):: 53.2894/55.4425  => accepted
04:37:52 - INFO     - cbt      - prefill/incerta04.front.sepia.ceph.com/1: iops_avg: (or (greater) (near 0.05)):: 13642.0/14193.0  => accepted
04:37:52 - INFO     - cbt      - prefill/incerta04.front.sepia.ceph.com/1: iops_stddev: (or (less) (near 0.50)):: 91.0293/207.767  => accepted
04:37:52 - INFO     - cbt      - prefill/incerta04.front.sepia.ceph.com/1: latency_avg: (or (less) (near 0.05)):: 0.00117061/0.00112533  => accepted
04:37:52 - INFO     - cbt      - rand/incerta04.front.sepia.ceph.com/0: bandwidth: (or (greater) (near 0.05)):: 255.714/248.44  => accepted
04:37:52 - INFO     - cbt      - rand/incerta04.front.sepia.ceph.com/0: iops_avg: (or (greater) (near 0.05)):: 65462.0/63600.0  => accepted
04:37:52 - WARNING  - cbt      - rand/incerta04.front.sepia.ceph.com/0: iops_stddev: (or (less) (near 0.50)):: 3830.51/2473.17  => rejected
04:37:52 - INFO     - cbt      - rand/incerta04.front.sepia.ceph.com/0: latency_avg: (or (less) (near 0.05)):: 0.000241491/0.000248547  => accepted
04:37:52 - WARNING  - cbt      - rand/incerta04.front.sepia.ceph.com/1: bandwidth: (or (greater) (near 0.05)):: 210.611/224.124  => rejected
04:37:52 - WARNING  - cbt      - rand/incerta04.front.sepia.ceph.com/1: iops_avg: (or (greater) (near 0.05)):: 53916.0/57375.0  => rejected
04:37:52 - INFO     - cbt      - rand/incerta04.front.sepia.ceph.com/1: iops_stddev: (or (less) (near 0.50)):: 1363.96/1299.68  => accepted
04:37:52 - WARNING  - cbt      - rand/incerta04.front.sepia.ceph.com/1: latency_avg: (or (less) (near 0.05)):: 0.000293595/0.000275694  => rejected
04:37:52 - WARNING  - cbt      - 4 tests failed out of 16
```

**Comment by tchaikov** [comment]
jenkins test classic perf

**Comment by tchaikov** [comment]
```
02:55:17 - INFO     - cbt      - prefill/incerta04.front.sepia.ceph.com/0: bandwidth: (or (greater) (near 0.05)):: 61.1565/55.2588  => accepted
02:55:17 - INFO     - cbt      - prefill/incerta04.front.sepia.ceph.com/0: iops_avg: (or (greater) (near 0.05)):: 15656.0/14146.0  => accepted
02:55:17 - INFO     - cbt      - prefill/incerta04.front.sepia.ceph.com/0: iops_stddev: (or (less) (near 0.50)):: 480.185/328.308  => accepted
02:55:17 - INFO     - cbt      - prefill/incerta04.front.sepia.ceph.com/0: latency_avg: (or (less) (near 0.05)):: 0.00101976/0.00112877  => accepted
02:55:17 - INFO     - cbt      - prefill/incerta04.front.sepia.ceph.com/1: bandwidth: (or (greater) (near 0.05)):: 60.5062/53.9999  => accepted
02:55:17 - INFO     - cbt      - prefill/incerta04.front.sepia.ceph.com/1: iops_avg: (or (greater) (near 0.05)):: 15489.0/13823.0  => accepted
02:55:17 - WARNING  - cbt      - prefill/incerta04.front.sepia.ceph.com/1: iops_stddev: (or (less) (near 0.50)):: 599.576/285.778  => rejected
02:55:17 - INFO     - cbt      - prefill/incerta04.front.sepia.ceph.com/1: latency_avg: (or (less) (near 0.05)):: 0.00103078/0.00115522  => accepted
02:55:17 - INFO     - cbt      - rand/incerta04.front.sepia.ceph.com/0: bandwidth: (or (greater) (near 0.05)):: 243.602/212.442  => accepted
02:55:17 - INFO     - cbt      - rand/incerta04.front.sepia.ceph.com/0: iops_avg: (or (greater) (near 0.05)):: 62362.0/54385.0  => accepted
02:55:17 - WARNING  - cbt      - rand/incerta04.front.sepia.ceph.com/0: iops_stddev: (or (less) (near 0.50)):: 4512.39/1150.6  => rejected
02:55:17 - INFO     - cbt      - rand/incerta04.front.sepia.ceph.com/0: latency_avg: (or (less) (near 0.05)):: 0.000253505/0.000290995  => accepted
02:55:17 - INFO     - cbt      - rand/incerta04.front.sepia.ceph.com/1: bandwidth: (or (greater) (near 0.05)):: 220.808/226.993  => accepted
02:55:17 - INFO     - cbt      - rand/incerta04.front.sepia.ceph.com/1: iops_avg: (or (greater) (near 0.05)):: 56526.0/58110.0  => accepted
02:55:17 - WARNING  - cbt      - rand/incerta04.front.sepia.ceph.com/1: iops_stddev: (or (less) (near 0.50)):: 2499.7/1593.35  => rejected
02:55:17 - INFO     - cbt      - rand/incerta04.front.sepia.ceph.com/1: latency_avg: (or (less) (near 0.05)):: 0.000279888/0.000272271  => accepted
02:55:17 - WARNING  - cbt      - 3 tests failed out of 16
```

**Comment by tchaikov** [comment]
changelog

- increase 0.5 to 2.

**Comment by tchaikov** [comment]
```
04:59:41 - DEBUG    - cbt      - Nodes : incerta04.front.sepia.ceph.com
04:59:41 - WARNING  - cbt      - prefill/incerta04.front.sepia.ceph.com/0: bandwidth: (or (greater) (near 0.05)):: 22.2215/31.5225  => rejected
04:59:41 - WARNING  - cbt      - prefill/incerta04.front.sepia.ceph.com/0: iops_avg: (or (greater) (near 0.05)):: 5688.0/8069.0  => rejected
04:59:41 - INFO     - cbt      - prefill/incerta04.front.sepia.ceph.com/0: iops_stddev: (or (less) (near 2.00)):: 109.33/139.507  => accepted
04:59:41 - WARNING  - cbt      - prefill/incerta04.front.sepia.ceph.com/0: latency_avg: (or (less) (near 0.05)):: 0.00280377/0.00196233  => rejected
04:59:41 - WARNING  - cbt      - prefill/incerta04.front.sepia.ceph.com/1: bandwidth: (or (greater) (near 0.05)):: 22.2509/31.2792  => rejected
04:59:41 - WARNING  - cbt      - prefill/incerta04.front.sepia.ceph.com/1: iops_avg: (or (greater) (near 0.05)):: 5696.0/8007.0  => rejected
04:59:41 - INFO     - cbt      - prefill/incerta04.front.sepia.ceph.com/1: iops_stddev: (or (less) (near 2.00)):: 86.1704/51.9615  => accepted
04:59:41 - WARNING  - cbt      - prefill/incerta04.front.sepia.ceph.com/1: latency_avg: (or (less) (near 0.05)):: 0.00279886/0.00199072  => rejected
04:59:42 - WARNING  - cbt      - rand/incerta04.front.sepia.ceph.com/0: bandwidth: (or (greater) (near 0.05)):: 167.302/240.522  => rejected
04:59:42 - WARNING  - cbt      - rand/incerta04.front.sepia.ceph.com/0: iops_avg: (or (greater) (near 0.05)):: 42829.0/61573.0  => rejected
04:59:42 - INFO     - cbt      - rand/incerta04.front.sepia.ceph.com/0: iops_stddev: (or (less) (near 2.00)):: 2356.81/1288.52  => accepted
04:59:42 - WARNING  - cbt      - rand/incerta04.front.sepia.ceph.com/0: latency_avg: (or (less) (near 0.05)):: 0.000370478/0.000256635  => rejected
04:59:42 - WARNING  - cbt      - rand/incerta04.front.sepia.ceph.com/1: bandwidth: (or (greater) (near 0.05)):: 166.884/243.3  => rejected
04:59:42 - WARNING  - cbt      - rand/incerta04.front.sepia.ceph.com/1: iops_avg: (or (greater) (near 0.05)):: 42722.0/62284.0  => rejected
04:59:42 - INFO     - cbt      - rand/incerta04.front.sepia.ceph.com/1: iops_stddev: (or (less) (near 2.00)):: 2157.1/1417.09  => accepted
04:59:42 - WARNING  - cbt      - rand/incerta04.front.sepia.ceph.com/1: latency_avg: (or (less) (near 0.05)):: 0.000371104/0.000253855  => rejected
04:59:42 - WARNING  - cbt      - 12 tests failed out of 16
```

**Comment by tchaikov** [comment]
jenkins test classic perf

**Comment by tchaikov** [comment]
```
05:34:18 - INFO     - cbt      - prefill/incerta04.front.sepia.ceph.com/0: bandwidth: (or (greater) (near 0.05)):: 55.3665/54.2704  => accepted
05:34:18 - INFO     - cbt      - prefill/incerta04.front.sepia.ceph.com/0: iops_avg: (or (greater) (near 0.05)):: 14173.0/13893.0  => accepted
05:34:18 - INFO     - cbt      - prefill/incerta04.front.sepia.ceph.com/0: iops_stddev: (or (less) (near 2.00)):: 207.29/315.644  => accepted
05:34:18 - INFO     - cbt      - prefill/incerta04.front.sepia.ceph.com/0: latency_avg: (or (less) (near 0.05)):: 0.00112668/0.00114944  => accepted
05:34:18 - INFO     - cbt      - prefill/incerta04.front.sepia.ceph.com/1: bandwidth: (or (greater) (near 0.05)):: 52.7619/53.3748  => accepted
05:34:18 - INFO     - cbt      - prefill/incerta04.front.sepia.ceph.com/1: iops_avg: (or (greater) (near 0.05)):: 13507.0/13663.0  => accepted
05:34:18 - INFO     - cbt      - prefill/incerta04.front.sepia.ceph.com/1: iops_stddev: (or (less) (near 2.00)):: 254.23/362.749  => accepted
05:34:18 - INFO     - cbt      - prefill/incerta04.front.sepia.ceph.com/1: latency_avg: (or (less) (near 0.05)):: 0.00118235/0.00116906  => accepted
05:34:18 - WARNING  - cbt      - rand/incerta04.front.sepia.ceph.com/0: bandwidth: (or (greater) (near 0.05)):: 218.112/254.126  => rejected
05:34:18 - WARNING  - cbt      - rand/incerta04.front.sepia.ceph.com/0: iops_avg: (or (greater) (near 0.05)):: 55836.0/65056.0  => rejected
05:34:18 - INFO     - cbt      - rand/incerta04.front.sepia.ceph.com/0: iops_stddev: (or (less) (near 2.00)):: 1398.7/4079.0  => accepted
05:34:18 - WARNING  - cbt      - rand/incerta04.front.sepia.ceph.com/0: latency_avg: (or (less) (near 0.05)):: 0.000283415/0.000242963  => rejected
05:34:18 - INFO     - cbt      - rand/incerta04.front.sepia.ceph.com/1: bandwidth: (or (greater) (near 0.05)):: 252.485/224.891  => accepted
05:34:18 - INFO     - cbt      - rand/incerta04.front.sepia.ceph.com/1: iops_avg: (or (greater) (near 0.05)):: 64636.0/57572.0  => accepted
05:34:18 - INFO     - cbt      - rand/incerta04.front.sepia.ceph.com/1: iops_stddev: (or (less) (near 2.00)):: 3572.26/3164.74  => accepted
05:34:18 - INFO     - cbt      - rand/incerta04.front.sepia.ceph.com/1: latency_avg: (or (less) (near 0.05)):: 0.00024459/0.000274599  => accepted
05:34:18 - WARNING  - cbt      - 3 tests failed out of 16
```

**Comment by tchaikov** [comment]
jenkins test classic perf

**Comment by tchaikov** [comment]
> > @markhpc i am not sure what the right approach is ... my biggest concern is that the performance could be degraded by a random PR which is merged because we are very tolerant. and when we narrow the the requirements down, it might be way too late.
> 
> That's possible, but it would still be a better situation than we have now as we could at least go back and look for trends in the performance data to narrow down when it happened even if it's noisy. I don't think we'll ever get to the point where we can avoid all false positives or false negatives. It's going to be a balancing act.

makes sense, Mark. we really need to make better use of the historical data . that's a better way to understand the trend and the behavior of each sha1 / run.

**Pull request merged**